src/coordinates/uniform_cartesian.hpp

```diff
@@ -277,13 +294,43 @@ class UniformCartesian {
     return 0.0;
   }
 
+  //----------------------------------------
+  // Cylindrical+Spherical Conversions
+  //----------------------------------------
+  // TODO
```

Comment:
longer discussion below. Resolving this comment

Reply:
From what I've done and what Shengtai included in his demo, there's a bigger use case to provide spherical and cylindrical conversions in Parthenon since we're likely to deal with physical systems where those coordinates make sense (i.e. galaxy clusters and stars for spherical setups, jets and disks for cylindrical setups). We could leave it up to each of these systems to implement these conversions on their own but I see them repeated so many times elsewhere it might be better to centralize that code in one place. 

I'm not set on including these conversions in Parthenon at the moment but still thinking about it. We might also include conversions for vectors.

Reply:
I see. No I think that makes sense. As I comment below, we may also want it for vis tools. It would be better if, for example, Paraview could automatically visualize simulations done in spherical coordinates. So we might want to standardize some functions that always map from whatever coordinate system we're in to Cartesian.